dizoo/classic_control/example/mtcar_rainbow.py

```diff
@@ -0,0 +1,13 @@
+from easydict import EasyDict
```

Comment:
remove this file and update it in #471

Reply:
Done with [df15717](https://github.com/opendilab/DI-engine/pull/452/commits/df15717300358de4acb3aab8010eea18c0e26f6f)